app/src/Homepage/Homepage-components.test.tsx

```diff
@@ -25,4 +25,14 @@ test('Renders LevelTab components with correct names', () => {
   expect(tab4).toBeInTheDocument();
 });
 
+test('GetWordAfterSpace returns string of text after space', () => {
```

Comment:
Your test has a different method than what is being tested. GetWordAfterSpace vs GetStringAfterSpace. 

I also added commented below to change the name though

Reply:
changed